src/main/java/com/denizenscript/depenizen/bukkit/commands/vivecraft/ViveMirrorCommand.java

```diff
@@ -0,0 +1,84 @@
+package com.denizenscript.depenizen.bukkit.commands.vivecraft;
+
+import com.denizenscript.denizen.objects.NPCTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import com.denizenscript.depenizen.bukkit.bridges.ViveCraftBridge;
+import com.denizenscript.depenizen.bukkit.objects.vivecraft.ViveCraftPlayerTag;
+import org.bukkit.entity.Player;
+import org.vivecraft.VSE;
+import org.vivecraft.VivePlayer;
+
+import java.util.List;
+
+
+public class ViveMirrorCommand extends AbstractCommand {
+
+    public ViveMirrorCommand() {
+        setName("vivemirror");
+        setSyntax("vivemirror [<npc>] [mirror:<vivecraftplayer>] (targets:{player}/<player>|...)");
+        setRequiredArguments(2, 3);
+        autoCompile();
+    }
+
+    // <--[command]
+    // @Name ViveMirror
+    // @Syntax vivemirror [<npc>] [mirror:<vivecraftplayer>] (targets:{player}/<player>|...)"
+    // @Required 1
```

Comment:
this is required `2` technically, unless the `mirror` argument should use the attached player in the queue as the default `vivecraftplayer``